portfolio/src/main/webapp/index.html

```diff
@@ -1,55 +1,44 @@
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta charset="utf-8">
     <title>About Kofi Meighan !</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
 
-
-
-  <body bgcolor ="AliceBlue">
+  <body bgcolor ="aliceblue">  
     <h1 align="center">Kofi Meighan</h1>
 
+    <!--Personal info links at the top left of the screen-->
     <ul class="info" align="left">
-        <li><a href="https://github.com/kofimeighan">GitHub</a></li>
-        <li><a href="https://www.linkedin.com/in/kofimeighan/">LinkedIn</a></li>
-        <li>(561) 613-1936</li>
-        <li>kofimeighan@google.com</li>
+      <li><a href="https://github.com/kofimeighan">GitHub</a></li>
+      <li><a href="https://www.linkedin.com/in/kofimeighan/">LinkedIn</a></li>
+      <li>(561) 613-1936</li>
+      <li><a href="mailto:kofimeighan@google.com">kofimeighan@google.com</a></li>
     </ul>
 
+    <!--Navigation bar at the top of the screen-->
     <ul class="nav">
-        <li><a href="education.html">Education & Activities</a></li>
-        <li><a href="pictures.html">Film Pictures</a></li>
-        <li><a href="mixes.html">Music Mixes</a></li>
+      <li><a href="index.html">Home</a></li>
+      <li><a href="education.html">Education & Activities</a></li>
+      <li><a href="pictures.html">Film Pictures</a></li>
+      <li><a href="mixes.html">Music Mixes</a></li>
     </ul>
 
     <h3><b>Who am I?</b></h3>
-    <p> My name is Kofi Meighan, and I am from Boynton Beach, Florida. I began 
-        my ventures within Computer Science when I was very young and I didn't 
-        even know it. I saw coding in action when I would program lego robots
-        to move a designated period of time, a set number of rotations, or a 
-        very precisely calculated number of radians. That was the start, and
-        finally being able to apply my skills to projects that will advance
-        technology and positively develop someone's experience.
-
-    <div class="row">
-        <div class="column">
-            <img src="images/pink!.jpg" style="width:100%">
-        </div>
-        <div class="column">
-            <img src="images/kofianddylan.jpg" style="width:100%">
-        </div>
-        <div class="column">
-            <img src="images/cups headshot.jpg" style="width:100%">
-        </div>
-    </div>
-
-    
-    
-          
-
+      <p> My name is Kofi Meighan, and I am from Boynton Beach, Florida. I 
+        began my ventures within Computer Science when I was very young and 
+        I didn't even know it. I saw coding in action when I would program 
+        lego robots to move a designated period of time, a set number of 
+        rotations, or a very precisely calculated number of radians. 
+        That was the start, and finally being able to apply my skills to 
+        projects that will advance technology and positively develop 
+        someone's experience.
+
+        <img src="images/kofianddylan.jpg" 
```

Comment:
There are three line wrapping methods in the style guide.  (https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping). 

Try not to mix them.

Reply:
I went through and applied a consistent wrapping method in all the files.